tests/phpunit/FileSystem/FileSystemTestCase.php

```diff
@@ -84,9 +86,16 @@ protected function tearDown(): void
 
     final protected static function removeTmpDir(): void
     {
+        $testToken = getenv('TEST_TOKEN');
+
         (new Filesystem())->remove(
             normalizePath(
-                realpath(sys_get_temp_dir()) . '/' . self::TMP_DIR_NAME
+                sprintf(
```

Comment:
when tests are executed on parallel, different threads started to remove the temp folders of another thread. We need to have different folders for each thread, which is done here